cmd/main_postsubmit.go

```diff
@@ -117,6 +121,13 @@ func main() {
 		"kubernetes-csi/external-snapshotter",
 		"kubernetes-csi/external-provisioner",
 	}
+
```

Comment:
```suggestion

# Changing the order here to try to ensure the projects (etcd, metrics-server, csi's etc) are built first. 
# All the release branches build in parallel, and since kubernetes takes a longer time, having the first version 
# build the other projects first, when the remaining release branches get around to building the other 
# projects, they'll have those images ready to pull and use for the layer cache if applicable. Using the 
# cached version, the projects that are the same regardless of release branches (i.e. metric-server)
# should not have to produce a new (duplicate) image and can tag the existing one if it's the same. 
```

Reply:
I wrote something similar in the makefile. do you want them both? would you rather over what i wrote?

Reply:
I guess it makes sense to just have it in the makefile